ssl/src/main/java/org/wildfly/security/ssl/MechanismDatabase.java

```diff
@@ -35,21 +35,32 @@
 import java.util.regex.Pattern;
 
 class MechanismDatabase {
-    private static final MechanismDatabase INSTANCE = new MechanismDatabase();
+    private static final MechanismDatabase INSTANCE = new MechanismDatabase("MechanismDatabase.properties");
+    private static final MechanismDatabase TLS13_INSTANCE = new MechanismDatabase("TLS13MechanismDatabase.properties", true);
```

Comment:
I wonder if these static references should be referenced from different classes?  That way unless they are both used they won't both be dynamically loaded.